packages/metrics/src/Metrics.ts

```diff
@@ -49,7 +49,7 @@ const DEFAULT_NAMESPACE = 'default_namespace';
  *
```

Comment:
Minor: [L38](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/996dba61795b526cbfc0b209119b781b80e3c488/packages/metrics/src/Metrics.ts#L38) of the comment has another occurrence of the word `raise`, could you please update that to `throw` too?